src/go/k8s/tests/e2e/scale-up/02-assert.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: scale-up-cluster
+status:
+  readyReplicas: 2
```

Comment:
Should we have additional e2e test that check the following assertion?
```suggestion
apiVersion: redpanda.vectorized.io/v1alpha1
kind: Cluster
metadata:
  name: scale-up-cluster
status:
  replicas: 2
```

Reply:
perhaps in another PR then?

Reply:
Ok thanks.